content/docs/command-reference/exp/apply.md

```diff
@@ -0,0 +1,138 @@
+# exp apply
+
+Apply the changes from a previously run experiment into the workspace.
+
+## Synopsis
+
+```usage
+usage: dvc exp apply [-h] [-q | -v] experiment
+
+positional arguments:
+  experiment     Experiment to be applied.
+```
+
+## Description
+
+This command will apply the changes from an experiment to the workspace. It is
```

Comment:
"from an existing experiment"?
"previous experiment"?
"experiment in {some term for the log or cache of exps}"?

Reply:
Also you can just start descriptions with the verb in 2nd person e.g. "Applies the changes..." — we tend to do that in cmd ref descs.